qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/utils/DocumentationUtils.java

```diff
@@ -340,14 +344,9 @@ private static void addUrl(String url, StringBuilder documentation, boolean mark
 		if (!StringUtils.isEmpty(url)) {
 			documentation.append(System.lineSeparator());
 			documentation.append("See ");
-			if (markdown) {
-				documentation.append("[here](");
-				documentation.append(url);
-				documentation.append(")");
-			} else {
-				documentation.append(url);
-			}
+			addLink(url, "here", documentation, markdown);
 			documentation.append(" for more informations.");

```

Comment:
This should be: `" for more information."` (i.e. no `s` at the end of information).

Reply:
fixed